micronaut-camunda-bpm-example/src/main/java/info/novatec/micronaut/camunda/bpm/example/rest/spi/ServletProcessEngineProvider.java

```diff
@@ -0,0 +1,25 @@
+package info.novatec.micronaut.camunda.bpm.example.rest.spi;
+
+import org.camunda.bpm.engine.ProcessEngine;
+import org.camunda.bpm.engine.ProcessEngines;
+import org.camunda.bpm.engine.rest.spi.ProcessEngineProvider;
+
+import java.util.Set;
+
+// File copied from Spring Boot Starter
```

Comment:
please add full link to Github

please add a javadoc

feel free to add yourself as author to the class